api/models/usersChannels.js

```diff
@@ -384,8 +382,7 @@ const approveBlockedUserInChannel = async (channelId: string, userId: string): P
 
   return db
     .table('usersChannels')
-    .getAll(channelId, { index: 'channelId' })
-    .filter({ userId, isBlocked: true })
+    .getAll([userId, channelId], { index: 'userIdAndChannelId' })
```

Comment:
Sorry, one more spot: I think we want to filter `isBlocked: true` to ensure that it's only approving a blocked user

Reply:
Committing